rope/base/project.py

```diff
@@ -459,7 +460,7 @@ def _get_file(self, name, compress):
             path += '.gz'
         return self.project.get_file(path)
 
-
+@utils.cached(1000)
 def _realpath(path):
```

Comment:
Yes, it hardest place in profiler. Very precision cache, it saves ~50% of time. We need cache in this place.